# StopAuction.srv - Service to stop the auction
---
bool success
string message